src/SME.SGP.Dados/Repositorios/RepositorioConsolidacaoRegistroIndividualMedia.cs

```diff
@@ -14,6 +16,29 @@ public RepositorioConsolidacaoRegistroIndividualMedia(ISgpContext database)
         {
             this.database = database;
         }
+
+        public async Task<long> Inserir(ConsolidacaoRegistroIndividualMedia consolidacao)
+        {
+            return (long)(await database.Conexao.InsertAsync(consolidacao));
+        }
+
+        public async Task LimparConsolidacaoMediaRegistrosIndividuaisPorAno(int anoLetivo)
+        {
+            var query = @" delete from consolidacao_registro_individual_media
+                           where turma_id in (
+                                               select 
+                                                    distinct
+                                                    ri.turma_id
+                                                from registro_individual ri 
+                                                inner join turma tu on ri.turma_id = tu.id 
+                                                where not ri.excluido 
+	                                                and tu.ano_letivo = @anoLetivo
+	                                                and tu.modalidade_codigo in (1,2)
```

Comment:
jogar os valores de 1 e 2 pra uma variavel fora da query.
Ex: var modalidadesInfaltil = new [] {Modalidade.InfantilPreEscola, Modalidade.InfantilCEI} fazendo cast desses valores pra um array de int. Facilita o rastreamento caso haja alguma mudança nas modalidades

Reply:
a modalidade 2 não vai mais existir, se for usar a constante fica esperto porque o segundo vai ser removido na história de agrupamento.
Também tem a limitação do dapper com constantes, o parametro passado para query tem que ser um array de int